READMEs/data-nfts-and-datatokens-flow.md

```diff
@@ -100,20 +100,8 @@ Congrats, you've created your first Ocean data NFT!
 In the same python console:
 ```python
 # Create ERC20 token related to the above NFT.
-from ocean_lib.web3_internal.constants import ZERO_ADDRESS
-
-erc20_token = erc721_nft.create_datatoken(
-    template_index=1, # default value
-    name="ERC20DT1",  # name for ERC20 token
-    symbol="ERC20DT1Symbol",  # symbol for ERC20 token
-    minter=alice_wallet.address,  # minter address
-    fee_manager=alice_wallet.address,  # fee manager for this ERC20 token
-    publish_market_order_fee_address=alice_wallet.address,  # publishing Market Address
-    publish_market_order_fee_token=ZERO_ADDRESS,  # publishing Market Fee Token
-    publish_market_order_fee_amount=0,
-    bytess=[b""],
-    from_wallet=alice_wallet
-)
+
+erc20_token = erc721_nft.create_datatoken("ERC20DT1", "ERC20DT1Symbol", from_wallet=alice_wallet)
```

Comment:
Better name & symbol: "Datatoken 1" & "DT1"

Also, for the `create_erc721_nft()` call above, I suggest name & symbol of "Data NFT 1" & "DN1".